src/plugins/legacy/polygonRoi/polygonEventFilter.cpp

```diff
@@ -199,6 +195,22 @@ bool polygonEventFilter::mouseMoveEvent(medAbstractView *view, QMouseEvent *mous
 {
     savedMousePosition[0] = mouseEvent->x()*QGuiApplication::screenAt(mouseEvent->globalPos())->devicePixelRatio();
     savedMousePosition[1] = (currentView->viewWidget()->height()-mouseEvent->y()-1)*QGuiApplication::screenAt(mouseEvent->globalPos())->devicePixelRatio();
+    currentView->viewWidget()->setCursor(Qt::OpenHandCursor);
```

Comment:
this line will be replaced by the if/else conditions below, so it's not needed. But you could put that line at ArrowCursor by default:

```c++
currentView->viewWidget()->setCursor(Qt::ArrowCursor);
if (activeManager)
{
        if ( activeManager->getMinimumDistanceFromNodesToMouse(savedMousePosition, false) < 10.)
        {
            currentView->viewWidget()->setCursor(Qt::OpenHandCursor);
        }
}
```